### PR TITLE
Update Cypress Tailwind setup

### PR DESCRIPTION
**Pull request by TomasSlama**
[FX-NNNN]

### Description

We need mui styles to be injected first in cypress to have the same setup as product. After we updated the style order in Cypress, some bugs appeared in PageSidebar and Carousel.

Also some screenshots were updated, but it looks like the old screenshots were wrong

### How to test

- [Temploy](https://picasso.toptal.net/fx-null-happo-browser-update)
- Check Happo screenshots

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- Annotate all `props` in component with documentation
- Create `examples` for component
- Ensure that deployed demo has expected results and good examples
- Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests ([visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md) included)

**Breaking change**

- codemod is created and showcased in the changeset
- test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

**Comment by TomasSlama** [comment]
@toptal-anvil ping reviewers

File: packages/base/Carousel/src/CarouselNavigation/CarouselNavigation.tsx [hunk 1]
```diff
@@ -48,6 +48,8 @@ const CarouselNavigation = ({
 }: Props) => {
   const classes = useStyles()
 
+  console.log({ hasArrows, hasDots })
```
[the remark this answers is not in the document]
Comment (TomasSlama) [reply packages/base/Carousel/src/CarouselNavigation/CarouselNavigation.tsx:51]: done 👍, thank you for noticing

**Comment by TomasSlama** [comment]
@toptal-anvil ping reviewers

**Review by dmaklygin — APPROVED** [review]
Checked the code. Looks good

File: packages/base/Page/src/PageTopBar/__snapshots__/test.tsx.snap [hunk 2]
```diff
@@ -54,7 +54,7 @@ exports[`Page.TopBar render with custom logo 1`] = `
                 >
                   <div>
                     <div
-                      class="bg-white shadow-0 transition-shadow duration-300 delay-0 PicassoDropdown-content PageHamburger-responsiveWrapperContent"
+                      class="bg-white shadow-0 transition-shadow duration-300 delay-0 PicassoDropdown-content !shadow-[inset_-1px_0px_0px_0px] !shadow-gray max-h-[calc(100vh-var(--header !bg-gray"
```
Comment (sashuk) [inline packages/base/Page/src/PageTopBar/__snapshots__/test.tsx.snap:57]: Is it a correct class name? Maybe I am missing something, but it seems like there are not enough closing parenthesis in `max-h-[calc(100vh-var(--header`

Comment (TomasSlama) [reply packages/base/Page/src/PageTopBar/__snapshots__/test.tsx.snap:57]: good catch!, let me look into it

Comment (TomasSlama) [reply packages/base/Page/src/PageTopBar/__snapshots__/test.tsx.snap:57]: @sashuk this is the original `!shadow-[inset_-1px_0px_0px_0px] !shadow-gray-200 max-h-[calc(100vh-var(--header-height,3.5rem))] !bg-gray-100`

it seems that jest for some reason removes everything after the second `-`

Comment (sashuk) [reply packages/base/Page/src/PageTopBar/__snapshots__/test.tsx.snap:57]: @TomasSlama Got it, thanks! If we are confident that the original is the one we need, then probably fine to proceed (still strange that Jest does this, though)

**Review by sashuk — APPROVED** [review]
Looks good!

File: packages/base/Page/src/PageTopBar/__snapshots__/test.tsx.snap (hunk 2 above)
Comment (TomasSlama) [reply packages/base/Page/src/PageTopBar/__snapshots__/test.tsx.snap:57]: it is done to not create new snapshots all the time for JSS. We are striping the numbers from `.jss-classname-99`

**Pull request merged**